src/dropdown/dropdown.scss

```diff
@@ -67,29 +55,33 @@ $height-small: 30px !default;
 }
 
 .Select.dropdown {
-  @include normalize;
-  @include height($height);
   box-sizing: border-box;
+
+  @include normalize();
```

Comment:
this is the same as writing `@include normalize` right? anyway, we should test this too to see if final result is the same as it was before